Library/Homebrew/extend/os/linux/extend/ENV/super.rb

```diff
@@ -46,15 +46,30 @@ def homebrew_extra_paths
 
   def homebrew_extra_isystem_paths
     paths = []
-    # Add paths for GCC headers when building against glibc@2.13 because we have to use -nostdinc.
-    if deps.any? { |d| d.name == "glibc@2.13" }
+    # Add paths for GCC headers when building against glibc@2.13 or glibc@2.17
```

Comment:
Maybe this should be all versioned glibcs (`glibc@.+`) rather than hardcoding a list here?